code/stats/scoring.cpp

```diff
@@ -1362,7 +1505,7 @@ float scoring_get_scale_factor()
 void scoring_bash_rank(player *pl,int rank)
 {	
 	// if this is an invalid rank, do nothing
-	if((rank < RANK_ENSIGN) || (rank > RANK_ADMIRAL)){
```

Comment:
Off-by-one error in the code change here.  The > should now be >=.

Reply:
fixed